frontend/src/components/dashboard/minesHomePage/Dashboard.js

```diff
@@ -110,6 +111,7 @@ export class Dashboard extends Component {
       this.props.fetchMineDisturbanceOptions();
       this.props.fetchMineCommodityOptions();
       this.props.fetchPartyRelationshipTypes();
+      this.props.fetchPermitStatusOptions();
       this.props.setOptionsLoaded();
     }
```

Comment:
Did you have any annoyances when adding another staticContentActionCreator?
I'm just wondering because I did when adding `ProvinceCodes`, and I'm thinking of creating a ticket to move all of these to the highest level parent, ie `App.js`

Reply:
Justin missed that he had to make changes in two files the first time, which confused him for 20 minutes or so. We talked about this before and it makes sense to do so I totally support creating a ticket for that!

Reply:
Yeah, as Ryan said I was confused by it for 15-20 until Ryan pointed our the two separate entry points issue. I would be all for removing the need to have this in two places.